components/clarinet-cli/src/frontend/cli.rs

```diff
@@ -1779,7 +1696,7 @@ fn display_deploy_hint() {
     );
     println!(
         "{}",
-        yellow!("Find more information on the devnet here: https://docs.hiro.so/clarinet/guides/how-to-run-integration-environment")
+        yellow!("Find more information on the DevNet here: https://docs.hiro.so/clarinet/guides/how-to-run-integration-environment")
```

Comment:
```suggestion
        yellow!("Find more information on the devnet here: https://docs.hiro.so/clarinet/guides/how-to-run-integration-environment")
```
I think that across most of our Hiro docs we just use `devnet`.